aws/headers.py

```diff
@@ -67,6 +67,7 @@
   'media-src': [
     "'self'",
     'data:',
+    'blob:',
```

Comment:
Shouldn't this be above data though to show some respect to the ABC?

Reply:
will sort them all..! Good point